### PR TITLE
>>>BLOCKED<<< Disabling transition recording, trap checks, stale hunts and convertibles, added hunt count, and unobtrusive alert

### PR DESCRIPTION
**Pull request by DevJackSmith**
I know i'm breaking rules here by having too many changes in one PR. apologies.
* adding unobtrusive mode option to horn notification (and making it from background page instead of mh page) - credit to @haoala with their permission and mh devs permission
* filtering out all entries that are as old or older than pre-hunt response ones
* filtering out all hunts if trap check is in the list of new entries
* filtering out all hunts if stage changed from pre-hunt to post-hunt
* updating all messages to start with "MHCT: ..." for clarity in the console.log

Still few issues:
* FF doesn't want to work on my local machine (notifications not showing up at all except sound)
* hunt_count is just a display for now, needs implementation on the back end and probably a logic to discard if there's two or more hunts.

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -741,7 +741,16 @@
 
         // Perform validations and stage corrections.
         fixLGLocations(message, user_pre, user_post, hunt);
+        const temp_message_post = message; // For transition check
```
Comment (tehhowch) [inline src/scripts/main.js:744]: This is a shallow reference; any changes to the properties of `message` are also made to `temp_message_post`.

Comment (DevJackSmith) [reply src/scripts/main.js:744]: thank you. Javascript never ceases to annoy me :)

File: src/scripts/main.js [hunk 2]
```diff
@@ -712,44 +710,64 @@
...
+            window.console.log({message: "MHCT: ", differences});
         }
 
+        // Find maximum entry id from pre_response
+        let max_old_entry_id = pre_response.page.journal.entries_string.match(/data-entry-id='(\d+)'/g);
+        if (!max_old_entry_id.length) {
+            max_old_entry_id = 0;
+        } else {
+            max_old_entry_id = max_old_entry_id.map(x => x.replace(/^data-entry-id='/, ''));
+            max_old_entry_id = max_old_entry_id.map(x => Number(x.replace("'", "")));
```
[the remark this answers is not in the document]
Comment (DevJackSmith) [reply src/scripts/main.js:722]: we only need to replace one. (there's only two and first one is already replaced).

**Comment by DevJackSmith** [comment]
BLOCKED REASON: need to merge https://github.com/m-h-c-t/mh-helper-extension/pull/184 pr first to make this one much simpler.

**Comment by DevJackSmith** [comment]
continued on new branch here: https://github.com/m-h-c-t/mh-helper-extension/pull/185

**Pull request merged**